test/unit/project-management/ios-app.spec.ts

```diff
@@ -43,7 +44,7 @@ describe('IosApp', () => {
   beforeEach(() => {
     mockApp = mocks.app();
     requestHandler = new ProjectManagementRequestHandler(mockApp);
-    iosApp = new IosApp(APP_ID, requestHandler);
+    iosApp = new IosAppImpl(APP_ID, requestHandler);
```

Comment:
Based on the discussion we had offline, let's alias this to `IosApp` at the import. That way rest of the test implementation doesn't have to change.